whisper/whisperv6/whisper.go

```diff
@@ -590,10 +590,7 @@ func (whisper *Whisper) Unsubscribe(id string) error {
 // network in the coming cycles.
 func (whisper *Whisper) Send(envelope *Envelope) error {
 	ok, err := whisper.add(envelope, false)
-	if err != nil {
-		return err
-	}
-	if !ok {
+	if err == nil && !ok {
```

Comment:
If you do that and the error is not nil, it would make sense to include it in the error string

Reply:
if error if not nil, it will be processed and reported.
only in case err == nil, we need to report any issues on the spot.